vagrant/puppet/cmf/manifests/infrastructure/storage.pp

```diff
@@ -0,0 +1,45 @@
+class cmf::infrastructure::storage($mysql, $jack) {
```

Comment:
rename to cmf::infrastructure::phpcr in order to be consistent with the phpcr sub-namespace